### ISSUE TITLE
VoiceOver: Add unread messages to account switch accessibility label

### ISSUE THREAD

**Issue by cyBerta** [deltachat/deltachat-ios#1722]
Currently the amount of unread messages is read out by VoiceOver just as a number, the context of the number is missing. With #1721 VoiceOver doesn't read it out at all.

It would be nice to append a String "X unread messages" to the accessibility label of the account switch avatar, if there are any unread messages.

**Comment by cyBerta** [comment]
we already have strings n_messages, which are also used in the Chat List to read out the unread messages per chat. I'm going to reuse these strings
